javascript/javascript1/week2/classwork/bank.js

```diff
@@ -0,0 +1,27 @@
+const balance = 1000;
+
+if (balance <= 0) {
+  console.log("Please deposit some money!");
+} else if (balance > 0 && balance <= 1000) {
+  console.log("Your balance is looking okay");
+} else if (balance > 1000 && balance <= 3000) {
+  console.log("Your balance is looking good");
+} else if (balance > 3000 && balance <= 10000) {
+  console.log("Your balance is fantastic");
+} else {
+  console.log("Your balance is Amazing");
+}
```

Comment:
Maybe you could simplify the comparison, by just checking if(balance < VALUE).
If statements only jump to the next if the previous comparison evaluate as false